ACS/ACS/asterics.ini

```diff
@@ -35,5 +35,6 @@ pathToPluginActivationTool=.\tools\AsTeRICS_PluginActivation.exe
 pathToLanguageFileCreationTool=.\tools\LanguageFileCreator.exe
 pathToMidiCreationTool=.\tools\Midi_TonescaleEditor.exe.exe
 useAppDataFolder=false
-acsOnlineHelpBaseURL=http://asterics.github.io/AsTeRICS/
-helpBrowserCommand=explorer
\ No newline at end of file
+ACSManualURL=https://www.asterics.eu/manuals/ACS/
```

Comment:
yes, but there will be a short interleaving period where it won't work, because we have to create a release.
But in between, someone can use asterics.ini to change the URL to e.g. 
```
http://asterics.github.io/asterics-docs/
```